packages/@react-spectrum/table/docs/Table.mdx

```diff
@@ -0,0 +1,619 @@
+      <Table aria-label="example async loading table" height="100%" width="100%">
+        <TableHeader columns={columns}>
+          {(column) => (
+            <Column>{column.name}</Column>
+          )}
+        </TableHeader>
+        <TableBody
+          items={list.items}
+          isLoading={list.isLoading}
+          onLoadMore={list.loadMore}>
+          {(item) => (
+            <Row key={item.name}>{(key) => <Cell>{item[key]}</Cell>}</Row>
+          )}
+        </TableBody>
+      </Table>
+    </Flex>
+  );
+}
+```
+
+## Events
+
+### Selection
+Table supports multiple selection modes. By default, selection is disabled, however this can be changed using the `selectionMode` prop.
+Use `defaultSelectedKeys` to provide a default set of selected rows. Note that the  value of the selected keys must match the `key` prop of the Row.
+
+The example below demonstrates how one would use `defaultSelectedKeys`.
+
+```tsx example
+<Flex height="size-2000" width="size-6000">
+  <Table aria-label="Example table with single selection (uncontrolled)" width="100%" height="100%" selectionMode="single" defaultSelectedKeys={['2']}>
+    <TableHeader>
+      <Column>Name</Column>
+      <Column>Type</Column>
+      <Column>Level</Column>
+    </TableHeader>
+    <TableBody>
+      <Row key="1">
+        <Cell>Charizard</Cell>
+        <Cell>Fire, Flying</Cell>
+        <Cell>67</Cell>
+      </Row>
+      <Row key="2">
+        <Cell>Blastoise</Cell>
+        <Cell>Water</Cell>
+        <Cell>56</Cell>
+      </Row>
+      <Row key="3">
+        <Cell>Venusaur</Cell>
+        <Cell>Grass, Poison</Cell>
+        <Cell>83</Cell>
+      </Row>
+      <Row key="4">
+        <Cell>Pikachu</Cell>
+        <Cell>Electric</Cell>
+        <Cell>100</Cell>
+      </Row>
+    </TableBody>
+  </Table>
+</Flex>
+```
+
+To programmatically control row selection, use the `selectedKeys` prop paired with the `onSelectionChange` callback. The `key` prop from the selected row will
+be passed into the callback when the row is pressed, allowing you to update `selectedKeys` accordingly.
+
+Here is how you would control selection for the above example.
+
+```tsx example export=true
+function PokemonTable(props) {
+  let columns = [
+    {name: 'Name', uid: 'name'},
+    {name: 'Type', uid: 'type'},
+    {name: 'Level', uid: 'level'}
+  ];
+
+  let rows = [
+    {id: '1', name: 'Charizard', type: 'Fire, Flying', level: '67'},
+    {id: '2', name: 'Blastoise', type: 'Water', level: '56'},
+    {id: '3', name: 'Venusaur', type: 'Grass, Poison', level: '83'},
+    {id: '4', name: 'Pikachu', type: 'Electric', level: '100'}
+  ];
+
+  let [selected, setSelected] = React.useState(new Set(['2']));
+
+  return (
+    <Flex height="size-2000" width="size-6000">
+      <Table aria-label="Example table with single selection (controlled)" width="100%" height="100%" selectionMode="single" selectedKeys={selected} onSelectionChange={setSelected} {...props}>
```

Comment:
instead of width/height, since it's in a flex, it'll automatically have align = stretch (removing need for height)
and then you can use flexGrow={1} so it doesn't need width

Reply:
Sounds good, added a maxWidth for most of the examples so they don't look so wonky